docs/methods/metrics/mean_squared_error.md

```diff
@@ -0,0 +1,27 @@
+# Mean Squared Error
+
+The Mean Squared Error (MSE) or Mean Squared Deviation (MSD) of an estimator measures the average of error squares i.e. the average squared difference between the estimated values and true value.
+## Parameters
+
+| Name          | Definition                                                                                  | Type            |
+| ------------- | ------------------------------------------------------------------------------------------- | ----------------|
+| Pred          | Takes a vector of predicted values                                                          | `int,double`    |
+| Actual        | Determines the step size at each iteration while trying to minimize the loss function       | `int,double`    |
```

Comment:
It was intended that you'll change the definition according to your usecase

Reply:
Done sir